test/common/packagelib.py

```diff
@@ -134,6 +136,13 @@ def setUp(self):
         # HACK: kpatch check sometimes complains that we don't set up a full repo in unrelated tests
         self.allow_browser_errors("Could not determine kpatch packages:.*repodata updates was not complete")
 
+    @functools.cached_property
+    def getSystemdUnitsInfo(self):
+        return json.loads(self.machine.execute("systemctl list-units -la --plain -o json"))
+
+    def getSystemdExistingUnits(self):
+        return [foo["unit"] for foo in self.getSystemdUnitsInfo]
```

Comment:
This is missing a `()` to actually call the method.

Reply:
> This is missing a `()` to actually call the method.

Hi @martinpitt it is intentional, because using `@functools.cached_property` https://www.geeksforgeeks.org/python-functools-cached_property/ to call it just once, so that it should be property not the method. Or do you prefer to call it everytime, so that I should remove this cached_property decorator?

Reply:
Ah, property magic, I missed that sorry!